nrf_wifi/README.rst

```diff
@@ -6,6 +6,163 @@ nRF70 low-level API documentation
 The nRF70 Series Wi-Fi driver provides a low-level API for use cases where the application needs to access the nRF70 Series device directly.
 This is typically intended for customers who want to use the nRF70 Series device on a different platform than Zephyr.
 
+.. contents::
+   :local:
+   :depth: 2
```

Comment:
``.. contents`` block must be placed immediately after the page title header, so before line 6.